src/utils/constants.ts

```diff
@@ -175,3 +184,340 @@ export const DEFAULT_FUND_MANAGMENT = {
+    '0xae646817e458c0be890b81e8d880206710e3c44e00000000000000000000039d',
+    '0xaef2c171dbe64b0c18977e16e70bfd29d4ee0256000000000000000000000351',
+    '0xbbf9d705b75f408cfcaee91da32966124d2c6f7d00000000000000000000047e',
+    '0xbd724eb087d4cc0f61a5fed1fffaf937937e14de000000000000000000000473',
+    '0xbe0f30217be1e981add883848d0773a86d2d2cd4000000000000000000000471',
+    '0xc46be4b8bb6b5a3d3120660efae9c5416318ed40000000000000000000000472',
+    '0xc69771058481551261709d8db44977e9afde645000010000000000000000042a',
+    '0xc6eee8cb7643ec2f05f46d569e9ec8ef8b41b389000000000000000000000475',
+    '0xcba9ff45cfb9ce238afde32b0148eb82cbe635620000000000000000000003fd',
+    '0xcf8b555b7754556cf2ac2165e77ee23ed8517d7900020000000000000000045e',
+    '0xd0dc20e6342db2de82692b8dc842301ff9121805000200000000000000000454',
+    '0xd3d5d45f4edf82ba0dfaf061d230766032a10e07000200000000000000000413',
+    '0xd6d20527c7b0669989ee082b9d3a1c63af742290000000000000000000000483',
+    '0xda1cd1711743e57dd57102e9e61b75f3587703da0000000000000000000003fc',
+    '0xe1fb90d0d3b47e551d494d7ebe8f209753526b0100000000000000000000034e',
+    '0xee02583596aee94cccb7e8ccd3921d955f17982a00000000000000000000040a',
+    '0xf984eb2b8a7ef780245a797a2fccd82f346409ca00000000000000000000034d',
+    '0xff8f84e8c87532af96aef5582ee451572233678b000200000000000000000478',
+    '0x054e7b0c73e1ee5aed6864fa511658fc2b54bcaa000000000000000000000015',
+    '0x3f1a2c4a3a751f6626bd90ef16e104f0772d4d6b00020000000000000000001b',
+    '0x7275c131b1f67e8b53b4691f92b0e35a4c1c6e22000000000000000000000010',
+    '0xa154009870e9b6431305f19b09f9cfd7284d4e7a000000000000000000000013',
+    '0xa1d14d922a575232066520eda11e27760946c991000000000000000000000012',
+    '0xa826a114b0c7db4d1ff4a4be845a78998c64564c000000000000000000000008',
+    '0xea67626e1f0b59e0d172a04f5702ef90bcdf440c00000000000000000000000f',
+    '0xeb496161099d45b3ea4892408ef745c6182eb56e00000000000000000000000e',
+    '0xece571847897fd61e764d455dc15cf1cd9de8d6f000000000000000000000014',
+    '0xed3e2f496cbcd8e212192fb8d1499842f04a0d19000000000000000000000009',
+    '0x02c9dcb975262719a61f9b40bdf0987ead9add3a000000000000000000000006',
+    '0x16c9a4d841e88e52b51936106010f27085a529ec00000000000000000000000c',
+    '0x32be2d0ddeaf3333501b24a28668ce373ba8e763000200000000000000000014',
+    '0x32f03464fdf909fdf3798f87ff3712b10c59bd86000000000000000000000005',
+    '0x4b718e0e2fea1da68b763cd50c446fba03ceb2ea00000000000000000000000b',
+    '0x68a69c596b3839023c0e08d09682314f582314e5000200000000000000000011',
+    '0x6f34a44fce1506352a171232163e7716dd073ade000200000000000000000015',
+    '0x9e2d87f904862671eb49cb358e74284762cc9f42000200000000000000000013',
+    '0xac4b72c01072a52b73ca71105504f1372efcce0d000000000000000000000003',
+    '0xbfd65c6160cfd638a85c645e6e6d8acac5dac935000000000000000000000004',
+    '0xe274c9deb6ed34cfe4130f8d0a8a948dea5bb28600000000000000000000000d',
+];
```

Comment:
any way to filter them out in the subgraph, or RPC?

Reply:
These get used in the Subgraph provider query to filter. Or do you mean by default at source?

Reply:
yes, at source, since we know the pool is bricked, we could set a flag somewhere that it's killed, is there something like that on a contract?